src/app/components/CourseDetailHeader/CourseDetailHeader.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import styles from "./CourseDetailHeader.module.css";
+import ClassBallDoubleDigit from "../../assets/icons/ClassBallDoubleDigit";
+import ClassBallNo from "../../assets/icons/ClassBallNo";
+import ClassBallSingleDigit from "../../assets/icons/ClassBallSingleDigit";
+import RoundImage from "../RoundImage/RoundImage";
+
+function CourseDetailHeader(): JSX.Element {
+  return (
+    <div className={styles.header}>
```

Comment:
If this is a course detail header, consider using a `header` element here (you can add headers to sections too).

Reply:
done! Thank you!